app/src/main/java/org/schulcloud/mobile/data/model/CurrentUser.java

```diff
@@ -15,6 +24,8 @@
     public String schoolId;
     public String displayName;
 
+    public RealmList<RealmString> permissions;
+
     public String get_id() {
```

Comment:
Why do you don't want that? Since you use Gson for JSON mapping you can simply use the `@SerializedName` annotation (see the [wiki](https://github.com/google/gson/blob/master/UserGuide.md#json-field-naming-support)).

And you already do it [here](https://github.com/schul-cloud/schulcloud-mobile-android/blob/45a6612454845b688429590086c0439c0c9a51a0/app/src/main/java/org/schulcloud/mobile/data/model/Homework.java#L21) and [here](https://github.com/schul-cloud/schulcloud-mobile-android/blob/c0ef6a412126b3fa0434acb3048d615b35a5ce1f/app/src/main/java/org/schulcloud/mobile/data/model/Event.java#L25).